internal/builders/gradle/collect_release_artifacts.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+mkdir release-files-for-slsa
+GRADLE_VERSION=$(./gradlew properties -q | grep "version:" | awk '{print $2}')
+
+# Move artifacts from the user-supplied artifact list
+ARTIFACT_LIST=\"${ARTIFACT_LIST}\"
+for i in ${ARTIFACT_LIST//,/ }
+do
```

Comment:
Trim whitespace in case users set something like "foo.jar , bar.jar"

```suggestion
do
    i="${i#"${i%%[![:space:]]*}"}" # trim leading whitespace                                                                                                                                                                                                                                                                                                                                         
    i="${i%"${i##*[![:space:]]}"}" # trim trailing whitespace
```